cli/cmd/runserver.go

```diff
@@ -74,7 +80,8 @@ func init() {
 
 	runserverCmd.Flags().Bool("enable-default-network-policy", false, "Whether to install a default network policy in namespaces")
 	runserverCmd.Flags().StringSlice("default-network-policy-excludes", []string{"kube-system"}, "List of namespaces where the default network policy should not be installed")
-	runserverCmd.Flags().String("default-network-policy-configmap", "kube-system:karydia-default-network-policy", "Configmap where to load the default network policy from, in the format <namespace>:<name>")
+	// replaced by KarydiaConfig - it's now possible to change this value via 'manifests/config.yml'
```

Comment:
Would refrain from commenting out code and commenting out-commented code, because changes are version controlled anyway.